packages/agent/src/agents/graphs/__tests__/interactive.spec.ts

```
@@ -0,0 +1,761 @@
+    HumanMessage,
+    SystemMessage,
+  };
+});
+
+jest.mock('@langchain/core/prompts', () => ({
+  MessagesPlaceholder: class {
+    constructor(public _name: string) {}
+  },
+  ChatPromptTemplate: class {
+    static fromMessages() {
+      return new (require('@langchain/core/prompts').ChatPromptTemplate)();
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)